curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-5
 
 # --description--
 
-Below your `.header` element, create a new `div` element and assign it a `class` of `"gallery"`. This `div` will act as a container for the gallery images.
+Below your `.header` element, create a new `div` element and assign it a `class` of `gallery`. This `div` will act as a container for the gallery images.
```

Comment:
I'm not sure we want this change - we often use quotes around literal values, but not always

Reply:
Maybe we should add some sort of style guide in the contribution guide to prevent such confusion.
But according to me, the consistency in quotations marks will help the user to follow the tutorial.

Reply:
A bit late to the discussion, but for some reference, there are some notes on "Formatting challenge text" here.

> References to code parts (i.e. function, method, or variable names) should be wrapped in ``` backticks. See example below:

So, this removal of double quotes aligns with our documentation.

https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=formatting-challenge-text